lib/ami_spec/server_spec.rb

```diff
@@ -40,6 +47,7 @@ def run
 
       RSpec.configuration.fail_fast = true if @debug
 
+
```

Comment:
whitespace?

Reply:
ooops. I've spent too long in javascript land, where prettier cleans up my mess for me ;)